src/js/services/opencap.service.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+(function() {
+  angular.module('bitcoincom.services').factory('opencapService', function($q, $http) {
+    function getAddress(alias) {
+      let aliasData = validateAlias(alias);
+      if (aliasData.username === '' || aliasData.domain === '') {
+        return $q((resolve, reject) => {
+          return reject('Invalid OpenCAP alias');
+        });
+      }
+
+      let deferred = $q.defer();
+      $http
+        .get(`https://dns.google.com/resolve?name=_opencap._tcp.${aliasData.domain}&type=SRV`)
+        .then(function(response) {
+          deferred.resolve(
+            parseSRV(response.data)
+              .then(data => getAddresses(alias, data.host, data.dnssec))
+              .catch(function(error) {
+                return $q((resolve, reject) => {
+                  reject(error);
+                });
+              })
+          );
+        })
+        .catch(function(response) {
+          deferred.reject('Couldn\'t find srv record for the provided domain');
+        });
+      return deferred.promise;
+    }
+
+    var parseSRV = function(respData) {
+      return $q((resolve, reject) => {
```

Comment:
Now :  `=>`  : lambda expression is not supported for old javascript engine.
Should be : 
```
function onQ() {
}
```